deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -566,15 +579,99 @@ groups:
           severity: "warning"
           type: "ceph_default"
       - alert: "CephDaemonSlowOps"
-        for: "30s"
-        expr: "ceph_daemon_health_metrics{type=\"SLOW_OPS\"} > 0"
-        labels:
-          severity: 'warning'
-          type: 'ceph_default'
         annotations:
-          summary: "{{ $labels.ceph_daemon }} operations are slow to complete"
           description: "{{ $labels.ceph_daemon }} operations are taking too long to process (complaint time exceeded)"
           documentation: "https://docs.ceph.com/en/latest/rados/operations/health-checks#slow-ops"
+          summary: "{{ $labels.ceph_daemon }} operations are slow to complete"
+        expr: "ceph_daemon_health_metrics{type=\"SLOW_OPS\"} > 0"
+        for: "30s"
+        labels:
+          severity: "warning"
+          type: "ceph_default"
+  - name: "hardware"
```

Comment:
If wonder if these hardware alerts don't apply to rook. But since they are based on a ceph health status, if they don't apply, then ceph shouldn't raise them, and they would be a no-op in these alerts. So either way, seems fine to  keep them here.

Reply:
You are right. All these hardware-related alerts are based on ceph health status, so if they don't apply, then ceph shouldn't raise them. 

I assumed that the hardware monitoring could be enabled even in rook-ceph deployment, https://docs.ceph.com/en/latest/hardware-monitoring. Then the alerts could be useful for such deployment. 

In other cases, they shouldn't hurt anything.